### DISCUSSION TITLE
hours into setup and i cant figure out why jellyseerr cant see my jellyfin library [SETUP]

### DISCUSSION THREAD

**Discussion by heymiguel** [seerr-team/seerr#587] [Q&A] [answered]
hello :(

3am here and i give up.

[Plex Scan]: Scan interrupted {"errorMessage":"connect ECONNREFUSED 127.0.0.1:32400"}

is one message, then i get bounced around to a error for method about something not being a library.

I'm on unraid, and im just using the default settings for setting up the container. 

![image](https://github.com/Fallenbagel/jellyseerr/assets/13340150/4862cfb1-8566-4263-9e98-6e0e9b3b0e21)

getting repeated errors "Forbidden" but i can totally login using my admin credentials. Have hoped over to the discord too and my searching there is fruitless. Hope someone can help.

**Comment by fallenbagel** [comment]
Because probably your mediaServerType in the settings.json changed. It's trying to connect to plex instead of jellyfin. Can you please hop onto discord I'll guide you through how to fix this.

You see its trying to connect to `:32400` and not `8096`. That's the clue

**Comment by heymiguel** [comment]
https://discord.com/channels/595508571135803403/913281697368395826/1180987625939673140

started a convo here! (and DM'ed ya)

**Reply by fallenbagel** [reply, marked as answer]
I meant to hop onto our discord. Jellyseerr discord. I can guide you. The issue is your mediaservertype changed to plex so you can't login with jellyfin

**Reply by heymiguel** [reply]
i have found the answer, thanks to your discord <3

Sorry again for mistaking IBRA === jellyfin discord

**Comment by ADrunkDillweed** [comment]
I love the internet. No matter what question or problem I have, I'm guaranteed to find a thread where someone else had the same problem, and then fixed it with no notes or comments after about what caused what or how to fix.

**Reply by fallenbagel** [reply]
> I love the internet. No matter what question or problem I have, I'm guaranteed to find a thread where someone else had the same problem, and then fixed it with no notes or comments after about what caused what or how to fix.

Well, here's the thread with the fix *and* the explanation:

Stop Seerr, delete the contents of your config folder, and restart. The config likely got into a bad state, and a fresh start clears it up.

That said, this isn't really the right place for support questions. GitHub is for storing source code, bug reports and feature tracking. It isn't really designed for support usecases. Our support channels exist for exactly this kind of thing as stated everywhere (docs, readme, and even this thread), and you'll get much better and timely help there.

**Reply by ADrunkDillweed** [reply]
Lol Bagel, this wasn't meant as a jibe or anything towards you or anyone. And I know git is mostly code/bugs blah blah. Just searching for my exact issue pulls up this thread and pretty much just this. It's just one of those things that bounces around in my head as I'm troubleshooting things that I find hilarious.  But thanks. The fix for me was the legacy auth setting in jellyfin, because whatever upgrade between the two broke something.
